src/content/docs/features/process.mdx

```diff
@@ -3,12 +3,101 @@ title: Process
 description: Access the current process.
 ---
 
-import Stub from '@components/Stub.astro';
 import PluginLinks from '@components/PluginLinks.astro';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
 
 <PluginLinks plugin="process" />
 
-<Stub>
-	Based on
-	https://github.com/tauri-apps/plugins-workspace/tree/v2/plugins/process
-</Stub>
+This plugin provides APIs to access the current process. To spawn child processes, see the [shell](/features/shell) plugin.
+
+## Setup
+
+Install the plugin-process to get started.
+
+<Tabs>
+    <TabItem label="Automatic">
+
+    1. Use your project's package manager to add the dependency:
+
+    <CommandTabs npm="npm run tauri add process"
+    yarn="yarn run tauri add process"
+    pnpm="pnpm tauri add process"
+    cargo="cargo tauri add process" />
+
+    2. Modify `lib.rs` to initialize the plugin:
+
+    {/* TODO: Revise once https://github.com/tauri-apps/tauri/issues/7696 is in */}
+
+    ```rust ins={6}
+    // lib.rs
+    #[cfg_attr(mobile, tauri::mobile_entry_point)]
+    pub fn run() {
+        tauri::Builder::default()
+            // Initialize the plugin
+            .plugin(tauri_plugin_process::init())
+            .run(tauri::generate_context!())
+            .expect("error while running tauri application");
+    }
+    ```
+
+    </TabItem>
+    <TabItem label="Manual">
+
+    1. Run `cargo add tauri-plugin-process` to add the plugin to the project's dependencies in `Cargo.toml`.
+
+    2. Modify `lib.rs` to initialize the plugin:
+
+    ```rust ins={6}
+    // lib.rs
+    #[cfg_attr(mobile, tauri::mobile_entry_point)]
+    pub fn run() {
+        tauri::Builder::default()
+            // Initialize the plugin
+            .plugin(tauri_plugin_process::init())
+            .run(tauri::generate_context!())
+            .expect("error while running tauri application");
+    }
+    ```
+
+    3. If you'd like to utilize the plugin in JavaScript then install the npm package as well:
+
+    <CommandTabs
+        npm="npm install @tauri-apps/plugin-process"
+        yarn="yarn add @tauri-apps/plugin-process"
+        pnpm="pnpm add @tauri-apps/plugin-process"
+    />
+    </TabItem>
+
+</Tabs>
+
+## Usage
+
+The process plugin is available in both JavaScript and Rust.
+
+<Tabs>
+<TabItem label="JavaScript">
+
+```js
+import { exit, relaunch } from '@tauri-apps/plugin-process';
+
+// exits the app with the given status code
+await exit(0);
+
+// restarts the app
+await relaunch();
+```
+
+</TabItem>
+<TabItem label="Rust">
+
+```rust
+// exits the app with the given status code
+app.exit(0);
```

Comment:
Should this show where `app` is defined/created?

Reply:
I pushed a commit, for reference here is the rust code context:
```rust
#[tauri::command]
fn _exit (app: tauri::AppHandle) {
    app.exit(0);
}
```

Reply:
Just did a tweak to link to the Rust docs for it.